.docker/api-gateway-nginx.conf

```diff
@@ -0,0 +1,19 @@
+server {
+    server_name localhost;
+    listen 80;
+
+    location = /favicon.ico { access_log off; log_not_found off; }
+    location = /robots.txt { access_log off; log_not_found off; }
+    location = /ping {
+        add_header Cache-Control "must-revalidate, no-store";
+        add_header Content-Type "text/plain; charset=utf-8";
```

Comment:
Any way to avoid the default `Content-Type: application/octet-stream` that is also sent?

Reply:
I thought this would overwrite it (we use it in `journal-formula` for example)

Reply:
Clients will use this value as it comes after, but weird to also see the default.